src/main/java/com/hm/achievement/lang/LanguageConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.hm.achievement.lang;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.exception.PluginLoadError;
+import com.hm.mcshared.file.CommentedYamlConfiguration;
+import org.bukkit.configuration.InvalidConfigurationException;
+
+import java.io.IOException;
+
+public class LanguageConfig extends CommentedYamlConfiguration {
```

Comment:
Is this class actually useful? I don't see much benefit compared to using _CommentedYamlConfiguration_ directly like for the two other configuration files. I feel like we're loosing consistency and adding complexity to the code.

Reply:
You're probably right, It was implemented before the static methods in `Lang` which made `getMsg` quite obsolete.